wizard/WizardHome.qml

```diff
@@ -153,9 +153,7 @@ Rectangle {
                     text: qsTr("Change language") + translationManager.emptyString
 
                     onClicked: {
-                        wizardController.skipModeSelection = true;
-                        wizardController.wizardState = 'wizardLanguage';
-                        languageSidebar.open();
+                        appWindow.toggleLanguageView();
```

Comment:
The settings > layout > change language button needs updating too.

Reply:
Thanks, looks like I did not commit this file.

Reply:
Thanks